code/obj/flock/furniture.dm

```diff
@@ -358,6 +362,45 @@
 		return
 	..()
 
+/obj/grille/flock/attackby(obj/item/W, mob/user) //override totally to prevent wirecutter snipping etc
```

Comment:
Instead of overriding the whole proc to remove a few behaviors I'd rather just see some vars on the subtype that block them on the parent proc

Reply:
That is a better way of doing it. willfix.